### ISSUE TITLE
a11y of poll should be checked

### ISSUE THREAD

**Issue by phillimorland** [liqd/a4-meinberlin#3740]
use screen reader to check poll app and ensure labelling makes sense

**Comment by sabinammm** [comment]
I tested this. Everything seems to be labelled and makes sense.
